lib/mapObjects/CQuest.h

```diff
@@ -36,7 +36,11 @@ class DLL_LINKAGE CQuest final
 		MISSION_RESOURCES = 7,
 		MISSION_HERO = 8,
 		MISSION_PLAYER = 9,
-		MISSION_KEYMASTER = 10
+		MISSION_HOTA_MULTI = 10,
+		// end of H3 missions
+		MISSION_KEYMASTER = 100,
```

Comment:
shouldn't be keymaster = 10, hota_multi = 100?

Reply:
No. 'MISSION_KEYMASTER' is our addition, seems to be used by AI so its value can be safely changed.

MISSION_HOTA_MULTI = 10 is constant embedded in H3M format, so it has to have fixed value